util/fusermount.c

```diff
@@ -1602,7 +1602,9 @@ int main(int argc, char *argv[])
 	   it forks and messes with the file descriptors. */
 
         max_fd = sysconf(_SC_OPEN_MAX);
-        for(int fd=3; fd<=max_fd; fd++) { if (fd!=cfd) close(fd); }
+        for(int fd=3; fd<=max_fd; fd++) { 
+		if (fd!=cfd) close(fd); 
+	}
```

Comment:
Still doesn't match - there are spaces missing. Like `for (` and `fd <= max_fd)`. Close on the next line... And please make sure you are using tabs - I don't see that in github.